### ISSUE TITLE
Fix mathjax expressions on the site

### ISSUE THREAD

**Issue by deemp** [objectionary/normalizer#222]
See the [docs](https://rust-lang.github.io/mdBook/format/mathjax.html).
